tools/bin/build_image.sh

```diff
@@ -18,34 +17,15 @@ assert_root
 
 cd "$PROJECT_DIR"
 
-function validate_dockerignore() {
-  excludes_all=$(grep -w '^\*$' .dockerignore || true)
-  excludes_except=$(grep -w '^!.*' .dockerignore || true)
-  if [ -n "$excludes_all" ] || [ -n "$excludes_except" ]; then
-    error "Cannot include exclusion exceptions when following symlinks. Please use an exclude pattern that doesn't use exclude-all (e.g: *) or exclude-except (e.g: !/some/pattern)"
-  fi
-}
-
 args=(
     -f "$DOCKERFILE"
     -t "$TAGGED_IMAGE"
     --iidfile "$ID_FILE"
 )
 
-if [ "$FOLLOW_SYMLINKS" == "true" ]; then
-  exclusions=()
-  if [ -f ".dockerignore" ]; then
-    validate_dockerignore
-    exclusions+=(--exclude-from .dockerignore)
-  fi
-  # Docker does not follow symlinks in the build context. So we create a tar of the directory, following symlinks, and provide the archive to Docker
-  # to use as the build context
-  tar cL "${exclusions[@]}" . | docker build - "${args[@]}"
+JDK_VERSION="${JDK_VERSION:-17.0.4}"
```

Comment:
Suggestion: update to latest openjdk patch 17.0.8

Reply:
Sure, but I'll do this in a separate PR, as this will invalidate a bunch of docker images, I believe.